### PR TITLE
Feature 162 migrate to expo

### LINKED ISSUE #162: migrate to expo

### PR DESCRIPTION
**Pull request by veikkahonkanen**
Resolves #162

### CODE REVIEW

File: reset-script.sh [hunk 1]
```diff
@@ -1,8 +0,0 @@
-watchman watch-del-all && \
-rm -rf /tmp/metro-*] && \
-rm -rf node_modules && \
-npm install && \
-cd android/app/ && \
-rm -rf debug.keystore && \
-keytool -genkey -v -keystore debug.keystore -storepass android -alias androiddebugkey -keypass android -keyalg RSA -keysize 2048 -validity 10000 && \
-cd ../../
```
Comment (veikkahonkanen) [inline reset-script.sh:8]: Re-write the script instead of deleting it

**Review by veikkahonkanen — COMMENTED** [review]
Do not delete the submodule

File: babel.config.js [hunk 2]
```diff
@@ -1,12 +1,19 @@
...
-        path: ".env",
-        systemVar: "overwrite"
+      // ["module:react-native-dotenv", {      
+      //   "moduleName": "process.env",      
+      //   "path": ".env",      
+      //   "blacklist": null,      
+      //   "whitelist": null,      
+      //   "safe": false,      
+      //   "allowUndefined": true  
+      // }],
```
Comment (Daniilmass) [inline babel.config.js:13]: Should it stay commented or can it be removed?

**Review by Daniilmass — APPROVED** [review]

File: babel.config.js (hunk 2 above)
Comment (Chi-Chu319) [reply babel.config.js:13]: I will check, probably needs to be removed...

**Pull request merged**